### DISCUSSION TITLE
Invited New Users do not receive email

### DISCUSSION THREAD

**Discussion by alessiobrown** [getredash/redash#7158] [Q&A] [answered]
Invited several new users (from different email domains) and emails are not received (also not in spam et al.).
I would be grateful for support to troubleshoot. Thank you

**Comment by justinclift** [comment]
Which version of Redash are you using?

**Reply by alessiobrown** [reply, marked as answer]
Version: 8.0.0

**Comment by aymaneMx** [comment]
I'm facing the same issue. @alessiobrown, were you able to resolve it? Or did you find an alternative solution to gain access for new users?

**Reply by aymaneMx** [reply]
Version: 9.0.0-beta (041b184d)
